test/e2e/apimachinery/aggregator.go

```diff
@@ -101,6 +109,10 @@ var _ = SIGDescribe("Aggregator", func() {
 		TestSampleAPIServer(f, aggrclient, imageutils.GetE2EImage(imageutils.APIServer))
```

Comment:
TestSampleAPIServer creates RBAC objects and fails if they cannot be written... that means RBAC API enablement is required to pass conformance?

Reply:
Looking at the [git timeline](https://github.com/kubernetes/kubernetes/blame/master/test/e2e/apimachinery/aggregator.go#L99-L101) it looks like it's been a requirement for around 2 years now.

Reply:
Since most clusters enable both the RBAC API and the RBAC authorizer, and it's the only built-in way to configure authorization rules dynamically, it's reasonable for e2e tests to speculatively make API calls to create RBAC roles/bindings for the permissions they require.

However, if we say RBAC is strictly optional, then we have to allow for clusters which 1) don't enable the RBAC API and 2) grant permissions to the e2e test user differently.

The way to allow for that is to tolerate the RBAC API not existing ... in other words, attempt to make the RBAC API calls but treat failures as non-fatal and continue the test, since the RBAC calls are not the point of the test.